includes/base/element-base.php

```diff
@@ -619,6 +619,47 @@ public static function on_import_replace_dynamic_content( $config, $map_old_new_
 		return $config;
 	}
 
+	/**
+	 * On import replace dynamic content.
+	 *
+	 * @since 3.8.0
+	 * @access public
+	 *
+	 * @param array $config
+	 * @param array $new_ids_map
+	 * @param array|null $controls
+	 *
+	 * @return array Element data.
+	 */
+	public static function updated_on_import_replace_dynamic_content( array $config, array $new_ids_map, $controls = null ) : array {
+		$tags_manager = Plugin::$instance->dynamic_tags;
+
+		if ( empty( $config['settings'] ) || empty( $config['settings'][ $tags_manager::DYNAMIC_SETTING_KEY ] ) ) {
+			return $config;
+		}
+
+		foreach ( $config['settings'][ $tags_manager::DYNAMIC_SETTING_KEY ] as $dynamic_name => $dynamic_value ) {
+			$tag_config = $tags_manager->tag_text_to_tag_data( $dynamic_value );
+			$tag_instance = $tags_manager->create_tag( $tag_config['id'], $tag_config['name'], $tag_config['settings'] );
+
+			if ( ! $tag_instance ) {
+				continue;
+			}
+
+			if ( ! method_exists( $tag_instance, 'updated_on_import_replace_dynamic_content' ) ) {
+				// on_import_replace_dynamic_content() is deprecated since 3.8.0
+				// TODO: Remove this check in the future.
+				$tag_config = $tag_instance->on_import_replace_dynamic_content( $tag_config, $new_ids_map );
```

Comment:
I think you should just pass 'map_old_new_post_ids' so it will work correctly.
Am I wrong?

Reply:
You are right. Changed.